lib/matplotlib/figure.py

```diff
@@ -2065,21 +2058,6 @@ def get_constrained_layout_pads(self, relative=False):
         """
         return self._parent.get_constrained_layout_pads(relative=relative)
 
-    def init_layoutgrid(self):
```

Comment:
This was moved to `_constrained_layout`  Not how there are no `_layoutgrids` attached to the figure any longer...

Reply:
Do you want to add a changelog note?

Reply:
Do we need one for private attributes?

Reply:
Nope.